terraform/modules/psn/main.tf

```diff
@@ -13,22 +13,49 @@ variable "subnet_ids" {
   description = "The ID of one or more subnets in which to create a network interface for the endpoint."
 }
 
-variable "security_group_ids" {
+variable "security_group_id" {
   type        = "list"
-  description = "The ID of one or more security groups to associate with the network interface."
+  description = "The security group to allow access to the PSN VPC Endpoint."
 }
 
 resource "aws_vpc_endpoint" "psn_service" {
   vpc_id            = "${var.vpc_id}"
   service_name      = "${var.vpc_endpoint}"
   vpc_endpoint_type = "Interface"
 
-  security_group_ids = ["${var.security_group_ids}"]
+  security_group_ids = ["${aws.security_group.psn_endpoint.id}"]
```

Comment:
```suggestion
  security_group_ids = ["${aws_security_group.psn_endpoint.id}"]
```

Reply:
it took me a good 30 seconds of staring at this before I spotted what the actual difference was 🙈